tests/testthat/test_repr_packageIQR.r

```diff
@@ -0,0 +1,29 @@
+context('reprs of packageIQR')
+
+options(stringsAsFactors = FALSE)
+
+test_that('repr_html out the right html code', {
+  x = vignette()
```

Comment:
and while you’re at it in this file. indentation settings are per project in Rstudio so it’ll cause pain if a file is indented differently